nucliadb_utils/src/nucliadb_utils/audit/stream.py

```diff
@@ -222,27 +300,46 @@ def report_resources(
             storage=Storage(resources=resources),
         )
 
-    async def visited(self, kbid: str, uuid: str, user: str, origin: str):
-        auditrequest = AuditRequest()
+    async def visited(
```

Comment:
Seems like some of the audit methods are in fact not async. Can you do a pass on all of them and see if we can turn some of those into sync calls?

Reply:
i refactored them all to be sync